.github/workflows/rust.yml

```diff
@@ -0,0 +1,53 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Setup sccache
+      if: github.event_name != 'release' && github.event_name != 'workflow_dispatch'
+      uses: mozilla-actions/sccache-action@v0.0.6
```

Comment:
The speedup is certainly insane like it builds it in 2 minutes instead of 10, yes it's for free. Cache is emptied automatically by GitHub

Reply:
and in cmake is not transparent, but that's why I use sccache flags, this makes sense, because it compiles poppler almost instantly, which we don't care much about since it's a dependency